test/performance/perf_test.lua

```diff
@@ -101,15 +101,19 @@ local row_name = {
     insert = 'insert',
```

Comment:
Regarding line 58: `alias = 's-2'` is used twice.

Reply:
Fixed in separate commit